counsel.el

```diff
@@ -4918,6 +4918,36 @@ An extra action allows to switch to the process buffer."
 ;; `counsel-slime-repl-history' within
 ;; `counsel--browse-history-action'.
 
+;;** `counsel-esh-dir-history'
+(declare-function eshell/cd "eshell")
```

Comment:
`eshell/cd` is defined in `em-dirs`, not `eshell`.

Reply:
You're totally right. It's fixed now.